src/styles/createPalette.js

```diff
@@ -8,7 +8,7 @@ import {
     pink,
     red,
     teal,
-} from '../colors';
+} from '../colors/index';
```

Comment:
Is `index` not assumed here?  Is this necessary?

Reply:
Yes, it is, because, in the parent folder exists these elements:
|church-management/client/src/
|---colors/
|-------index.js
|---colors.js
So, if we don specify the index.js inside colors folder, createPalette.js assumes/imports colors.js and the library breaks D:

Reply:
Got it.  This is gross IMHO, but if it is what it is, we can live with it.  @War777 @morethanfire any thoughts?  Could we improve this somehow?  Or is this just fine?

Reply:
Yes, no need for `./src/colors.js` from what I am seeing. Please use the theme to grab the colors. @War777, let me know if you have any trouble.

Reply:
done!